pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -23,6 +23,8 @@ import (
 	"net/url"
 	"strconv"
 	"strings"
+
+	"golang.org/x/net/http/httpguts"
```

Comment:
looking at package docs and noticed.

> This package is shared by the standard library (which vendors it) and x/net/http2. It comes with no API stability promise. 

let's surface the fact that any update in httpguts can be a breaking change.

Reply:
We have quite a number of packages that fit that fit that description. The whole space under `golang.org/x` for example, as well as BBE.